### ISSUE TITLE
Resistance to induced hypothermia

### ISSUE THREAD

**Issue by mnk77** [mgijax/mammalian-phenotype-ontology#2629]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
Resistance to induced hypothermia

>2. Synonyms (e.g., absent spleen)
Resistance to hypothermia

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Higher than normal body temperature following induction of hypothermia by cold-exposure, dietary restriction, ethanol treatment, etc..

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
See PMID 24531262, 24072708, 24162845

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
abnormal adaptive thermogenesis MP:0011019

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
MGI:mperry

**Comment by anna-anagnostop** [comment]
I am wondering whether we should split the proposed resistance term into more specific terms to account for dietary restriction-, cold-, ethanol- and pharmacologically-induced hypothermia

**Comment by mnk77** [comment]
I don’t have a good idea of how many annotations these terms would apply to. I’m reviewing some 70 papers on impaired adaptive thermogenesis and have about 50 more papers to do through. Dietary restriction and co9ld induced hypothermia are the most common models that I’ve come across.

**Comment by sbello** [comment]
I've added the term 
decreased susceptibility to induced hypothermia
MP:0020523
We can always split off more specific subterms if needed
One lingering question should we have a term for 'enhanced adaptive thermogenesis' with this term as a child? We do have a term for impaired adaptive thermogenesis, but, none of the other terms dealing with abnormal responses to hyper- or hypotherma are under this term.

**Comment by sbello** [comment]
consider adding a term for increased susceptibility to induced hypothermia
